client/lib/mixins/update-post-status/README.md

```diff
@@ -1,36 +0,0 @@
-update-post-status
```

Comment:
Would it be worth making some amendments to this doc and moving it along side `components/update-post-status/index.js` at this point?